tests/complex/data/rhevm/rhev3_api.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<api>
+    <link href="/ovirt-engine/api/capabilities" rel="capabilities"/>
```

Comment:
This is xml taken directly from a rhev 3.6 installation.  It will listen on the older url of $hostname/api, but the 'official' url is $hostname/ovirt-engine/api